lint.go

```diff
@@ -1159,11 +1159,17 @@ func (f *file) lintErrors() {
 	}
 }
 
-func lintCapAndPunct(s string) (isCap, isPunct bool) {
+func lintCapAndPunct(s string) (isCap, isPunct, isNewline bool) {
 	first, firstN := utf8.DecodeRuneInString(s)
 	last, _ := utf8.DecodeLastRuneInString(s)
 	isPunct = last == '.' || last == ':' || last == '!'
+	isNewline = last == '\n'
 	isCap = unicode.IsUpper(first)
+	if isNewline && strings.Index(s, "\n") != len(s)-1 {
+		// Don't flag strings that contain more than one newline. They
```

Comment:
I think that would be a good qualification for this condition. Also, if you are at it, could you please write down how many instances do you find in general with this new check, and how many you found in the std and in the x packages.
I still do not have contributor rights for this project, so the merge decision is up to however is the current maintainer, but you have my vote especially since I made the same check for our linter ;-)